.github/workflows/push_docker.yaml

```diff
@@ -31,33 +40,32 @@ jobs:
           sudo rm -rf "/usr/local/share/boost"
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
       - name: Checkout repository
-        uses: actions/checkout@v3
-      - name: Docker Hub login
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKER_HUB_USERNAME }}
-          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+        uses: actions/checkout@v4
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
       - name: Generate image metadata
         id: meta
-        uses: docker/metadata-action@v4
+        uses: docker/metadata-action@v5
         with:
           images: |
             symbioticlab/zeus
             mlenergy/zeus
           tags: |
             type=ref,event=branch
             type=ref,event=tag
-      - name: Setup Docker Buildx
-        uses: docker/setup-buildx-action@v2
+            type=schedule
+            type=sha
```

Comment:
What do these do?

Reply:
It generates tags based on commit SHA and scheduled workflow like '2024-07-30', but just a unique way to identify docker images.

Reply:
Hmm I'm not understanding why we need these new additional tags

Reply:
I agree it's unnecessary. Removed!